src/dvclive/live.py

```diff
@@ -316,7 +323,7 @@ def log_param(self, name: str, val: ParamLike):
         """Saves the given parameter value to yaml"""
         self.log_params({name: val})
 
-    def log_artifact(self, path: StrPath):
+    def log_artifact(self, path: StrPath, **kwargs):
```

Comment:
Why not include `type` and `name` as explicit kwargs?

Reply:
And maybe even `desc` and `labels`?

Reply:
I added explicit names. The reasons were that it was less code and `type` is a reserved keyword in Python, but neither are strong reasons